### ISSUE TITLE
Data split

### ISSUE THREAD

**Issue by Pari-singh** [garrickbrazil/M3D-RPN#37]
Hi @garrickbrazil,

Thanks for making this work public. I wanted to understand more about the data split, why are they split in particular way (i.e., if they are following any guidelines etc) and are there 2 versions of data split?

Would really appreciate if you could provide the reason for choosing smaller training dataset than validation set (3682 training & 3799 val)

Thanks
